app/application/services/system_audit_service.py

```
@@ -0,0 +1,28 @@
+
+class SystemAuditService:
+    """Emits audit events to the event store. Full implementation in Phase 5."""
+
+    def __init__(self, db: AsyncSession) -> None:
+        self.db = db
+
+    async def emit_audit_event(
+        self,
+        tenant_id: str,
+        entity_type: str,
+        action: Any,
```

Ruff (0.14.14)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `action`

(ANN401)